tasks/main.yaml

```diff
@@ -232,3 +232,17 @@
     state: absent
   tags:
     - kafka_cleanup
+
+- name: logrotate | Upload logrotate configuration
+  template:
+    src: kafka_logrotate.j2
+    dest: /etc/logrotate.d/kafka
+    mode: 0640
+
+- name: logrotate | Create cron task for faster rotation checks
```

Comment:
I doubt a cron job is necessary here unless we need to ensure logrotate is triggered multiple times daily under heavy workloads (such as pgpool case)

Reply:
kafka create new log every hour, the goal was to reduce the constraint on filebeat with a purge every 6 hours.
We can keep it daily